docs/agent-dna-attestations.md

```
@@ -0,0 +1,123 @@
+
+Records are **append-only**. No updates after creation.
+
+## EAS Onchain Attestations
+**Chain:** Base mainnet (chain id 8453).  
+**Contracts (Base):**
+- Schema Registry: `0x4200000000000000000000000000000000000020`
+- EAS: `0x4200000000000000000000000000000000000021`
+
+### Schema
+Schema string (EAS):
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)